test/unit/auth/tenant.spec.ts

```diff
@@ -105,7 +121,7 @@ describe('Tenant', () => {
           .to.deep.equal(tenantOptionsServerRequest);
       });
 
-      it('should return the expected server request with multi-factor and phone config', () => {
+      it('should return the expected server request with multi-factor, phone and provider config', () => {
```

Comment:
nit: multi-factor and provider config are part of the same "multiFactorConfig" struct. So maybe call this:

"'should return the expected server request with multi-factor (SMS, TOTP) and testPhoneNumber config".

I assume thats what phone config means here.